app/views/users/show.html.erb

```diff
@@ -8,5 +8,14 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3><%= pluralize @user.microposts.count, t(".micropost") %></h3>
```

Comment:
Chỗ này sử dụng  Cách thể hiện số ít và số nhiều với i18n đi bạn https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0